scipy/diff/numdifftools/_step_generators.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import division
+import numpy as np
+EPS = np.finfo(float).eps
+
+
+def _generate_step(
```

Comment:
Actually, the parameters were violating the falke8 standards of having 79 characters per line.